Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -225,10 +225,18 @@ public override string ToString()
 
     public class Wheel
     {
+        [IonField]
+        internal string specification;
+
+        public Wheel()
+        {
+            this.specification = "default";
+        }
+
         [IonConstructor]
         public Wheel([IonPropertyName("specification")] string specification)
```

Comment:
Can we make this private?

Reply:
No because the constructor is not recognized in `targetType.GetConstructors()` if it's made private.